docker-compose.yaml

```diff
@@ -20,6 +21,11 @@ services:
       - discovery.type=single-node
       - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
     mem_limit: 1GB
+    healthcheck:
```

Comment:
does this make dependent services not start until health is green?

Reply:
Not unless they say to: https://github.com/liatrio/grafeas-elasticsearch/pull/26/files#diff-3fde9d1a396e140fefc7676e1bd237d67b6864552b6f45af1ebcc27bcd0bb6e9R12-R14